docs/hub/model-card-annotated.md

```diff
@@ -182,7 +182,7 @@ _Detail throughput, start/end time, checkpoint sizes, etc._
 
 # Evaluation
 
-**Section Overview:** This section describes the evaluation protocols, what is being measured in the evaluation, and provides the results.  Evaluation is ideally constructed with factors, such as domain and demographic subgroup, and metrics, such as accuracy, which are prioritized in light of foreseeable error contexts and groups. Target fairness metrics should be decided based on which errors are more likely to be problematic in light of the model use. You can also specify your model's evaluation results in a structured way in the model card metadata. Results are parsed by the Hub and displayed in a widget on the model page. See https://huggingface.co/docs/hub/model-cards#evaluation-results.
+**Section Overview:** This section describes the evaluation protocols, what is being measured in the evaluation, and provides the results.  Evaluation ideally has at least two parts, with one part looking at general performance ([Testing Data, Factors & Metrics](#testing-data-factors--metrics)) and another looking at performance with respect to specific social safety issues ([Societal Impact Assessment](#societal-impact-assessment-optional)), such as may be done with red-teaming. The former is ideally constructed with factors, such as domain and demographic subgroup, and metrics, such as accuracy, which are prioritized in light of foreseeable error contexts and groups. Target fairness metrics should be decided based on which errors are more likely to be problematic in light of the model use. You can also specify your model's evaluation results in a structured way in the model card metadata. Results are parsed by the Hub and displayed in a widget on the model page. See https://huggingface.co/docs/hub/model-cards#evaluation-results.
```

Comment:
Everything after `The former` only applies to the first group, including the structured reporting of `evaluation results`, which could be interpreted as being comprehensive of societal impact / red-teaming results (while reading it I was trying to parse what would be applicable to the Societal Impact Assessment topic). We could add a final comment about the Societal Impact Assessment being free text, but it's probably not necessary as it's explained later. Maybe use `quantitative evaluation results`?

Reply:
Gotcha, yes. What about moving the instructions for each into each corresponding section? Let me give it a whirl.